src/app.js

```diff
@@ -1,14 +1,14 @@
 import "./styles/main.scss";
 
 const navSlide = () => {
-  const burger = document.querySelector(".burger");
-  const nav = document.querySelector(".nav-links");
-  const navLinks = document.querySelectorAll(".nav-links li");
+  const burger = document.querySelector(".nav__burger");
```

Comment:
You should identify elements like that by id. Ids must be unique, classes do not.

`<div id="nav-burger">`

and then 

`const burger = document.querySelector("#nav-burger");`